event_handlers.go

```diff
@@ -0,0 +1,100 @@
+package lightstep
+
+import (
+	"log"
+	"sync"
+	"sync/atomic"
+)
+
+func init() {
+	OnEvent(NewOnEventLogOneError())
+}
+
+var eventHandler atomic.Value
+var eventHandlerLock sync.Mutex
+
+// An OnEventHandler can be registered with OnEvent to
+type OnEventHandler func(Event)
+
+// onEvent is a thread-safe function for emiting tracer events.
+func onEvent(event Event) {
```

Comment:
nit: It might be nice to add

```
if event == nil {
  return
}
```

So that callers don't necessary have to check for nil.